### DISCUSSION TITLE
Alert does not appear on screen

### DISCUSSION THREAD

**Discussion by rafaelfs98** [mantinedev/mantine#4212] [Q&A]
I can't get the alert component to appear on the screen, can anyone help me?

` const onSubmit = async (form: ServicosData) => {
    const { error } = await upsertServicos(form, Number(servicoId));

    if (!error) {
      return (
        <Alert
          icon={<IconAlertCircle size="1rem" />}
          title="Sucess!"
          color="green"
        >
          Sucess
        </Alert>
      );
    }

    return onError(error.message);
  };`

**Comment by bastiengrignon** [comment]
Your **Alert** component doesn't seems to be in a `.jsx` or `.tsx` file, I think for what you want to achieve you'll need to use Mantine notifications
